packages/components/src/List/ListComposition/DisplayMode/ListDisplayMode.component.js

```diff
@@ -68,18 +19,13 @@ function ListDisplayMode({ children, displayModesOptions, id, onChange, selected
 		return children;
 	}
 	return (
-		<div className={theme('tc-display-mode-toggle')}>
-			{displayModesOptions.map(displayModeOption => (
-				<DisplayModeIcon
-					displayMode={selectedDisplayMode || displayMode}
-					displayModeOption={displayModeOption}
-					icon={displayModeOption === 'table' ? 'talend-table' : 'talend-expanded'}
-					id={id}
-					key={displayModeOption}
-					onSelect={onSelect}
-				/>
-			))}
-		</div>
+		<DisplayModeToggle
+			id={id}
+			mode={selectedDisplayMode || displayMode}
+			displayModes={displayModesOptions}
+			onChange={(event, value) => onSelect(event, value)}
```

Comment:
```suggestion
			onChange={onSelect}
```

Reply:
I had the `jsx-no-bind` linter error https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md
But I transformed function declaration to make it pass :)

Reply:
I had the `jsx-no-bind` linter error https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-no-bind.md
But I transformed function declaration to make it pass :)